### DISCUSSION TITLE
Get assist information from entity

### DISCUSSION THREAD

**Discussion by peteh** [zachowj/node-red-contrib-home-assistant-websocket#937] [Feature Request]
Not sure if this is somehow possible but I'd like to either filter based on "assist enabled" in the getentitities node or be able to filter based on other meta information in a function node. 

Currently it does not seem possible to get the information if a node is enabled in assist or not. 

My use case: I use node red to pull all entities from homeassistant and then make a list for rhasspy that i send to it via rest api request and retrain it. 

As I accidently turned of my washing machine during a washer cycle as it's connected via a smart plug, I'd like to exclude certain entities from the list based on assist enabled or not.

**Comment by zachowj** [comment]
https://github.com/home-assistant/core/blob/1173039392d7e53e00774448939fbc096d5995c9/homeassistant/components/homeassistant/exposed_entities.py#LL425C32-L425C64

Looks like that endpoint gives you a list of entities with an exposure boolean. You can use the API node to request the data.

```
{"type": "homeassistant/expose_entity/list"}
```

**Reply by peteh** [reply]
Thanks that worked, I used the list to filter my entities and now I can filter out what is visible to my custom rhasspy extension.
